webserv.conf

```diff
@@ -1,63 +1,63 @@
-http {
-	types {
-		text/html   			html;
-		text/css    			css;
-		application/javascript	js;
-		image/jpeg				jpg;
-		image/png				png;
-		image/gif				gif;
-		application/pdf			pdf;
-		text/plain				txt;
-		application/xml			xml;
-		application/json		json;
-		video/mp4				mp4;
-		audio/mpeg				mp3;
-	}
+types {
```

Comment:
I checked the changed part. I will ensure that this is reflected in the request or mime type.